lib/components/Input/index.tsx

```diff
@@ -4,9 +4,10 @@ import HelperMessage from "./HelperMessage";
 import "./Input.scss";
 
 export type InputVariants = "general" | "success" | "error";
-interface InputProps
+export interface InputProps
```

Comment:
you're not importing it anywhere so there is no need to export it 🙏 
```suggestion
 interface InputProps
```